TTHAnalysis/macros/leptons/friendChunkCheck.sh

```diff
@@ -14,7 +14,7 @@ if [[ "$1" == "-z" ]]; then
 fi;
 
 what=$1
-for F in $(ls ${what}_*.chunk*.root | sed 's/\.chunk[0-9]\+//' | sort | uniq); do
+for F in $(ls ${what}*.chunk*.root | sed 's/\.chunk[0-9]\+//' | sort | uniq); do
     FILES=$(ls ${F/.root/.chunk*.root} | \
```

Comment:
Why is removing the underscore necessary? Can't you just give "evVarFriend" instead of "evVarFriend_" as an argument?